pkg/controllers/user/monitoring/operatorHandler.go

```diff
@@ -36,12 +37,18 @@ func (h *operatorHandler) sync(key string, obj *mgmtv3.Cluster) (runtime.Object,
 		}
 		newCluster = newObj.(*mgmtv3.Cluster)
 	} else { // should withdraw
+		areAllOwnedProjectMonitoringDisabling, err := allOwnedProjectsMonitoringDisabling(h.cattleProjectClient)
```

Comment:
Maybe we can combine all the logic into a `sync()` function like we did for cluster/project alert. And all the logic will be all together.